src/error.rs

```diff
@@ -370,6 +370,16 @@ impl Error {
             _ => None,
         }
     }
+
+    /// Per the CLAM spec, for sensitive commands we must redact everything besides the
+    /// error labels, error code, and error code name from errors received in response to
+    /// sensitive commands. Currently, the only field besides those that we expose is the
+    /// error message.
+    pub(crate) fn redact(&mut self) {
```

Comment:
you can find the ongoing thread about error redaction [here](https://github.com/mongodb/specifications/pull/1303#discussion_r973265187). 
I think we have consensus on what the redaction behavior should be (only redact command errors, and only redact fields that are not code, codename, and labels) but not necessarily consensus yet on the testing approach for it.

Reply:
Makes sense, thanks for the explanation!